build.gradle

```diff
@@ -12,7 +12,7 @@
 apply from: file("${rootDir}/build-resources.gradle")
 allprojects {
     group = 'com.amazon'
-    version = '1.0.0.0-beta1'
+    version = '1.0.0.0-rc1'
```

Comment:
We might stick with `1.0.0-rc1` for Data Prepper, I don't think we need to follow the 4-digit OpenSearch version system.